lib/matplotlib/lines.py

```diff
@@ -622,23 +622,29 @@ def recache(self, always=False):
         if len(x) != len(y):
             raise RuntimeError('xdata and ydata must be the same length')
 
-        x = x.reshape((len(x), 1))
-        y = y.reshape((len(y), 1))
+        self._xy = np.empty((len(x), 2), dtype=np.float_)
```

Comment:
As it was this _sometimes_ returns a view, sometimes a copy, now this always copies.  Do we care about that change?